packages/core/android/src/newarch/java/io/sentry/react/replay/RNSentryReplayUnmaskManager.java

```diff
@@ -0,0 +1,36 @@
+package io.sentry.react.replay;
+
+import androidx.annotation.NonNull;
+import com.facebook.react.module.annotations.ReactModule;
+import com.facebook.react.uimanager.ThemedReactContext;
+import com.facebook.react.uimanager.ViewGroupManager;
+import com.facebook.react.uimanager.ViewManagerDelegate;
+import com.facebook.react.viewmanagers.RNSentryReplayUnmaskManagerDelegate;
+import com.facebook.react.viewmanagers.RNSentryReplayUnmaskManagerInterface;
+
+@ReactModule(name = RNSentryReplayUnmaskManager.REACT_CLASS)
+public class RNSentryReplayUnmaskManager extends ViewGroupManager<RNSentryReplayUnmask>
+    implements RNSentryReplayUnmaskManagerInterface<RNSentryReplayUnmask> {
+  private final RNSentryReplayUnmaskManagerDelegate<
+          RNSentryReplayUnmask, RNSentryReplayUnmaskManager>
+      delegate = new RNSentryReplayUnmaskManagerDelegate<>(this);
+
+  @Override
+  public ViewManagerDelegate<RNSentryReplayUnmask> getDelegate() {
+    return delegate;
+  }
+
+  @NonNull
+  @Override
+  public String getName() {
+    return REACT_CLASS;
+  }
+
+  @NonNull
+  @Override
+  public RNSentryReplayUnmask createViewInstance(@NonNull ThemedReactContext context) {
+    return new RNSentryReplayUnmask(context);
+  }
+
+  public static final String REACT_CLASS = "RNSentryReplayUnmask";
```

Comment:
q: Could we reuse `RNSentryReplayUnmaskManagerImpl.REACT_CLASS` here?

Reply:
Yes, definitely, thank you, I've missed this place, for mask it's already done.